plugins/jcanvas-handles.js

```diff
@@ -109,10 +109,31 @@ function addRectHandle( $canvas, parent, px, py ) {
 		drag: function ( layer ) {
 			var parent = layer._parent;
 			
+			if (parent.width + layer.dx * layer._px < parent.minWidth) {
```

Comment:
I realize that JavaScript's operator precedence will make this irrelevant, but for clarity, I would prefer that you wrap your multiplication in parens like so:

```javascript
if (parent.width + (layer.dx * layer._px) < parent.minWidth) {
```